README.md

```diff
@@ -59,8 +59,8 @@ versions. The next table shows the version compatability between GraphDB, and th
 
 | GraphDB Terraform                                                              | GraphDB                                                                              |
 |--------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
-| [Version 1.x.x](https://github.com/Ontotext-AD/terraform-aws-graphdb/releases) | [Version 10.6.x](https://graphdb.ontotext.com/documentation/10.6/release-notes.html) |
-| [Version 1.2.x](https://github.com/Ontotext-AD/terraform-aws-graphdb/releases) | [Version 10.7.x](https://graphdb.ontotext.com/documentation/10.7/release-notes.html) |
+| [Version 1.x.x](https://github.com/Ontotext-AD/terraform-aws-graphdb/releases) | [Version 10.7.x](https://graphdb.ontotext.com/documentation/10.6/release-notes.html) |
+| [Version 1.2.x](https://github.com/Ontotext-AD/terraform-aws-graphdb/releases) | [Version 10.8.x](https://graphdb.ontotext.com/documentation/10.8/release-notes.html) |
```

Comment:
AFAIK there are no breaking changes in GDB that require this update in the compatibility matrix.